bedrock/contentful/templates/includes/contentful/all.html

```diff
@@ -134,10 +134,12 @@ <h2 class="mzp-c-section-heading">{{ entry.heading }}</h2>
         mobile_class=entry.mobile_class,
```

Comment:
**issue (blocking):** we seem to have lost the mobile centering class (side-by-side screenshot in comment below, new split component is on right)

(I wish I had a suggestion for this but I don't know why it's happening, was there a change to the mobile display field in the content model?)

<img width="232" alt="Screen Shot 2022-06-07 at 12 00 11 PM" src="https://user-images.githubusercontent.com/19650432/172365381-d1456700-84e6-48f2-b0a9-ba4664ff19c9.png">

Reply:
There was a typo in `api.py` which is why this wasn't working correctly. This PR should fix this.